src/backend/distributed/citus.sql

```diff
@@ -349,8 +378,18 @@ BEGIN
 
         -- delete partition entry
         DELETE FROM pg_dist_partition WHERE logicalrelid = v_obj.objid;
-
     END LOOP;
+
+	IF cardinality(sequence_names) = 0 THEN
+		RETURN;
+	END IF;
+
+	FOR node_name, node_port IN
+	SELECT DISTINCT name, port
+	FROM unnest(node_names, node_ports) AS nodes(name, port)
+	LOOP
+		PERFORM master_drop_sequences(sequence_names, node_name, node_port);
+	END LOOP;
 END;
 $cdbdt$;
 COMMENT ON FUNCTION citus_drop_trigger()
```

Comment:
We concatenate all upgrades to make the "final" install script, right? I suppose that could work.

I _kind of_ like having the "canonical" install script actually be a long-lived file in the repository (to track minor changes to functions and whatnot) as opposed to an artifact built of all the upgrade scripts, but I suppose we're not firmly in either camp right now (some of each).